pilot/prompts/development/parse_task.prompt

```diff
@@ -1,2 +1,4 @@
-Ok, now, take your previous message and convert it to actionable items. An item might be a code change or a command run. When you need to change code, make sure that you put the entire content of the file in the value of `content` key even though you will likely copy and paste the most of the previous message. The commands must be able to run on a {{ os }} machine.
-{{ list_running_processes }}
+Ok, now, take your previous message and convert it to actionable items. An item might be a code change or a command run. When you need to change code, make sure that you put the entire content of the file in the value of `content` key even though you will likely copy and paste the most of the previous message.

```

Comment:
why did you remove "The commands must be able to run on a {{ os }} machine."? we need to provide OS to LLM so he can give correct command to run